docs/cli/overview.md

```diff
@@ -0,0 +1,101 @@
+
+- Installs or verifies that Aspires local hosting certificates are installed and trusted.
+- Builds the apphost project.
+- Starts the apphost.
+- Starts the dashboard.
+- Displays a terminal-based dashboard.
+
+<!-- Add asciinema here instead of the terminal dashboard -->
+
+The following snippet is an example of the terminal dashboard:
+
+```Aspire CLI
+Dashboard:
+📈  Direct: https://localhost:17077/login?t=64ebc6d760ab2c48df93607fd431cf0b
+
+╭─────────────┬─────────┬─────────┬────────────────────────╮
+│ Resource    │ Type    │ State   │ Endpoint(s)            │
+├─────────────┼─────────┼─────────┼────────────────────────┤
+│ apiservice  │ Project │ Running │ https://localhost:7422 │
+│             │         │         │ http://localhost:5327  │
+│ webfrontend │ Project │ Running │ https://localhost:7241 │
+│             │         │         │ http://localhost:5185  │
+╰─────────────┴─────────┴─────────┴────────────────────────╯
+Press CTRL-C to stop the apphost and exit.
+```
+
+## Add integrations
+
+The `aspire add` command is an easy way to add official integration packages to your apphost. Use this as an alternative to a NuGet search through your IDE. You can run `aspire add <name|id>` if you know the name or NuGet ID of the integration package, If you omit a name or ID, the tool provides a list of packages to choose from. If you provide a partial name or ID, the tool filters the list of packages with items that match the provided value.
+
+<!-- Add asciinema here -->
+
+## Publish Aspire applications (preview)
+
+The `aspire publish` command publishes resources, serializing data into JSON format. When this command is run, Aspire invokes registered <xref:Aspire.Hosting.ApplicationModel.PublishingCallbackAnnotation> annotations for resources. These annotations serialize a resource so that it can be consumed by deployment tools.
+
+## Deploy Aspire applications (preview)
+
+The `aspire deploy` command is similar to `aspire publish`, except that in addition to serializing resources, it also deploys resources that have registered a `DeployingCallbackAnnotation` annotation.
+
+> [!TIP]
+> Consider this a good way to deploy your Aspire solution to a staging area for testing.
+
+## Run commands in resource context (preview)
+
+The `aspire exec` command runs commands in the context of a specific Aspire resource, inheriting that resource's configuration including environment variables, connection strings, and working directory. This is particularly useful for scenarios like running Entity Framework migrations where you need to run commands with the same configuration as your application. For example, you can run `aspire exec --resource api -- dotnet ef migrations add Init` to run Entity Framework commands with the proper database connection strings automatically configured.
```

Comment:
Does it make sense to have links to the command docs under each section that talks through it conceptually?